cli/whoami.ts

```diff
@@ -0,0 +1,37 @@
+import type {ArgumentsCamelCase} from 'yargs';
+import {createCodaClient} from './helpers';
+import {formatEndpoint} from './helpers';
+import {getApiKey} from './config_storage';
+import {isResponseError} from '../helpers/external-api/coda';
+import {printAndExit} from '../testing/helpers';
+import {tryParseSystemError} from './errors';
+
+interface WhoamiArgs {
+  apiToken?: string;
+  codaApiEndpoint: string;
+}
+
+export async function handleWhoami({apiToken, codaApiEndpoint}: ArgumentsCamelCase<WhoamiArgs>) {
+  const formattedEndpoint = formatEndpoint(codaApiEndpoint);
+  if (!apiToken) {
+    apiToken = getApiKey(codaApiEndpoint);
+    if (!apiToken) {
+      return printAndExit('Missing API key. Please run `coda register` to register one.');
```

Comment:
Nit, let's be consistent about "key" vs "token" in developer-facing language, so would use "token" here.

Reply:
Updated here and for all the other places this error appears (create, link, release, upload)